app/api/odm/model.ts

```diff
@@ -32,8 +31,16 @@ class UpdateLogHelper {
   }
 
   upsertLogOne(doc: mongoose.Document, next: (err?: mongoose.NativeError) => void) {
+    // temporary prevent creating updateLogs without mongoid, log error
+    if (!doc._id) {
+      errorLog.error(
+        `ERROR, Update Log received a doc without _id:\n ${JSON.stringify(doc, null, ' ')}`
```

Comment:
Should we include here the collection name as well, to help us debug from where is this coming?  Or are we sure they are always entities?

Reply:
@RafaPolit we do not know what doc is in that case, all updatelogs without mongoId i have found are entities.